app/assets/javascripts/comments.js

```diff
@@ -255,10 +256,10 @@ $(() => {
 });
 
 
-window.new_thread = function (post_id){
+const newThread = (postId) => {
     const thread = document.getElementById("new_thread_modal_"+post_id);
 
-    if (thread.style.display=="none"){
+    if (thread.style.display == "none"){
```

Comment:
Always use `===` unless you're comparing to `null` or `undefined`

Reply:
Actually, the problem is I never formatted code. :( Sorry!